sdk/communication/Azure.ResourceManager.Communication/CHANGELOG.md

```diff
@@ -1,16 +1,20 @@
 # Release History
 
-## 1.1.0-beta.7 (Unreleased)
+## 1.1.0 (2023-03-31)
 
 ### Features Added
+- Added SenderUsernameResource, SenderUsernameResourceCollection and SenderUsernameResourceData to support the new resource type.
 
 ### Breaking Changes
+- Removed ValidSenderUsernames property from CommunicationServiceResourceData.
+- CommunicationResource RegenerateKey and RegenerateKeyAsync are no longer marked as long running operations.
+- CommunicationResource Update and UpdateAsync are no longer marked as long running operations.
 
 ### Bugs Fixed
 
 ### Other Changes
 
-## 1.1.0-beta.6 (2023-02-14)
+## 1.1.0-beta.6 (2023-02-14)S
```

Comment:
nit: not sure what the `S` is at the end of this line?

Reply:
Resolved.